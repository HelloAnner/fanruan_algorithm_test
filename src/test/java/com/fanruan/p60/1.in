3 3 2
8657835 6388365 4406730 
9596574 1765206 1740440 
1054683 1122743 5835610 
