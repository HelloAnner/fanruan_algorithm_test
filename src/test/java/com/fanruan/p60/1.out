7201853 7226619 